expression/builtin_math.go

```diff
@@ -843,38 +844,36 @@ func (b *builtinAtanSig) eval(row []types.Datum) (d types.Datum, err error) {
 	return d, nil
 }
 
+//cos start
```

Comment:
There is no other place in this file has this kind of comment, so I suggest that this can be removed.